modules/apps/portal-workflow/portal-workflow-kaleo-runtime-impl/src/main/java/com/liferay/portal/workflow/kaleo/runtime/internal/notification/TemplateNotificationMessageGenerator.java

```diff
@@ -112,7 +113,7 @@ private Template _getTemplate(
 		return TemplateManagerUtil.getTemplate(
 			templateManagerName,
 			new StringTemplateResource(templateId, notificationTemplate),
-			false);
+			PropsValues.NOTIFICATION_EMAIL_TEMPLATE_ENABLED);
```

Comment:
This property [has the default value false for LXC (SaaS)](https://github.com/liferay/liferay-portal/blob/2a9932479dabeda057a4a38b43f9969f243c2560/portal-impl/src/portal-liferay-online-config.properties#L12), so the correct thing to do is to negate the return value when passing like [here](https://github.com/liferay/liferay-portal/blob/af7cc75e11c044bd5abe9a9aaebf33a45584ee01/modules/apps/notification/notification-service/src/main/java/com/liferay/notification/internal/type/EmailNotificationType.java#L556).

Reply:
Since you are changing this class, would it be possible to write a test to cover this gap?